src/vs/base/browser/ui/positronComponents/splitters/verticalSplitter.tsx

```diff
@@ -376,6 +252,116 @@ export const VerticalSplitter = ({
 		setHighlightExpandCollapse(false);
 	};
 
+	/**
+	 * pointerDown handler.
+	 * @param e A PointerEvent that describes a user interaction with the pointer.
+	 */
+	const sashPointerDownHandler = (e: React.PointerEvent<HTMLDivElement>) => {
+		// Ignore events we don't process.
+		if (e.pointerType === 'mouse' && e.buttons !== 1) {
+			return;
+		}
+
+		// Determine whether the event occurred inside the expand / collapse button. If it did,
+		// don't process the event.
+		if (isPointInsideElement(e.clientX, e.clientY, expandCollapseButtonRef.current)) {
+			return;
+		}
+
+		// Consume the event.
+		e.preventDefault();
+		e.stopPropagation();
+
+		// Setup the resize state.
+		const resizeParams = onBeginResize();
+		const startingWidth = collapsed ? sashWidth : resizeParams.columnsWidth;
+		const target = DOM.getWindow(e.currentTarget).document.body;
```

Comment:
My first inclination is that the whole window body is too broad, but you're adding the events in a way that they don't clobber anything else listening... so this seems good to me!

Reply:
Hah, great point! 😄 I had the same exact inclination initially as well and I chatted about this exact thing with @softwarenerd. My understanding is that we need to know when a user does a drag and a release that ends outside of the application window. I'm a little new to handling drag events and followed the existing pattern we have in `<HorizontalSplitter>`.